charts/ceph-csi-cephfs/values.yaml

```diff
@@ -171,7 +171,7 @@ provisioner:
   provisioner:
     image:
       repository: registry.k8s.io/sig-storage/csi-provisioner
-      tag: v3.2.1
+      tag: v3.3.0
```

Comment:
in the commit message you mention version 3.3.1. Which is the correct one?

Reply:
3.3.0 is the correct one. corrected the commit message